.github/workflows/windows.yml

```diff
@@ -15,10 +15,9 @@ jobs:
         run: choco install 7zip wget python
       - name: Get snap7
         run: |
-          wget.exe --content-disposition -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
+          wget.exe -O snap7-full-1.4.2.7z -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
           7z x snap7-full-1.4.2.7z
-          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll snap7\.
-          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll C:\Windows\System32\.
+          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll C:\Windows\System32\
```

Comment:
we copy the dll into the folder to prepare for the binary wheel.

Reply:
Are you sure it works correctly? If snap7 folder doesn't exist then a file with that name will be created. Sorry, I probably don't fully understand.
![copy-item-test](https://user-images.githubusercontent.com/52915342/100585233-eaa52580-32f5-11eb-9fae-ceee558da13f.png)

Reply:
this is executed in the python-snap7 folder, right? so a `snap7` module should exist.

Reply:
Now I get it(and feel stupid). Thanks for the explanation. I'll move it back.

Reply:
no worries, you are still being very useful :)